pyaerocom/helpers.py

```diff
@@ -1682,3 +1684,75 @@ def get_time_rng_constraint(start, stop):
     t_upper = iris.time.PartialDateTime(year=stop.year, month=stop.month, day=stop.day)
 
     return iris.Constraint(time=lambda cell: t_lower <= cell <= t_upper)
+
+
+def get_max_period_range(periods):
+    start = min([int(per.split("-")[0]) for per in periods])
+    stop = max(
+        [int(per.split("-")[1]) if len(per.split("-")) > 1 else int(per) for per in periods]
+    )
+
+    return start, stop
+
+
+def _make_dummy_cube(
+    var_name: str, start_yr: int = 2000, stop_yr: int = 2020, freq: str = "daily", dtype=float
+) -> iris.cube.Cube:
```

Comment:
same as before, if this is a private function?
if so, why it is not called on this module?